services/user.py

```diff
@@ -0,0 +1,55 @@
+from django.contrib.auth import get_user_model
+from django.db.models import QuerySet
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> QuerySet:
+    new_user = get_user_model().objects.create_user(
+        username=username, password=password
+    )
+
+    if email:
+        new_user.email = email
+    if first_name:
+        new_user.first_name = first_name
+    if last_name:
+        new_user.last_name = last_name
```

Comment:
You can move this code to separate function and remove duplicate of code from create_user and update_user